roles/zabbix_agent/tasks/Debian.yml

```diff
@@ -199,8 +199,7 @@
   when: ansible_distribution in ['Ubuntu', 'Debian']
   register: zabbix_agent_package_installed
   until: zabbix_agent_package_installed is succeeded
-  become: true
-  check_mode: false
+  become: yes
```

Comment:
Why did you change this to `yes` while `true`|`false` is used everywhere in this project? 🤔